alfasim_sdk/types.py

```diff
@@ -227,15 +230,15 @@ def check(  # pylint: disable=arguments-differ
             raise TypeError(f"{attr.name} must be a Quantity, got a {type(values)}.")
 
 
-@attr.s(kw_only=True)
+@attr.s(kw_only=True, frozen=True)
 class Table(BaseField):
-    rows: List[TableColumn] = attrib()
+    rows: FrozenSet[TableColumn] = attrib(converter=frozenset)
 
     @rows.validator
     def check(  # pylint: disable=arguments-differ
         self, attr: Attribute, values: Union[List[str], str]
     ):
-        if not isinstance(values, list):
+        if not isinstance(values, frozenset):
```

Comment:
Doesn't the converter ensure this will always be true?

Reply:
Yep, this check is redundant I will remove it =)